plutus-chain-index-core/notes/ConnectingToTheTestnet.md

```diff
@@ -14,7 +14,7 @@ You need to download the configurations for the node, genesis blocks and the top
 I am using a shell script to start the node that I will paste here:
 
 ```shell
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
In the future if you see a quick fix that is possible create a new PR for it as there are many more people who can review and accept it.

Reply:
Yup, already did #618! (As that one is now merged, then can rebase it now and the commit won't be part of this PR anymore)